### PR TITLE
Cleaning up some guard checks.

### PR DESCRIPTION
**Pull request by biscuitWizard**
All of these checks are things that tripped up persistence game mode. Either they are too hostile to parameterless construction, or their null checks were not good enough.

Updated for friendliness.

### CODE REVIEW

**Comment by afterthought2** [comment]
Historically `to_world_log` and similar has been pretty useless, as it's hard to parse for debug information while simultaneously spamming admins. One construction that works for me is `#ifdef UNIT_TEST CRASH`, but I'm not certain others are as into it as I am.

**Comment by biscuitWizard** [comment]
> Historically `to_world_log` and similar has been pretty useless, as it's hard to parse for debug information while simultaneously spamming admins. One construction that works for me is `#ifdef UNIT_TEST CRASH`, but I'm not certain others are as into it as I am.

I'm okay looking at alternatives, but crashing is unacceptable. I need to be able to construct things with parameterless New() downstream.

File: code/datums/wires/wires.dm [hunk 1]
```diff
@@ -27,7 +26,12 @@ var/list/wireColours = list("red", "blue", "green", "darkred", "orange", "brown"
 	..()
 	src.holder = holder
 	if(!istype(holder, holder_type))
+#ifdef DISABLE_DEBUG_CRASH
+		to_world_log("Our holder is null/the wrong type!")
```
Comment (MistakeNot4892) [inline code/datums/wires/wires.dm]: CRASH_WITH supplies debugging info (stack trace), this just prints a sourceless message without saying who 'our' refers to.

Comment (biscuitWizard) [reply code/datums/wires/wires.dm]: we're pro engineers. we don't need stack traces. honestly I was tempted to remove the to_world_log entirely.

Comment (MistakeNot4892) [reply code/datums/wires/wires.dm]: That would be preferrable to the current line.

**Review by comma — APPROVED** [review]
